lexos/static/js/scripts_similarity.js

```diff
@@ -12,8 +12,9 @@ function runModal (htmlMsg) {
  * @returns {string | null} the errors that is checked by JS, if there is no error the result will be null
  */
 function submissionError () {
+    const active_file_num_too_few_err = 'You do not have enough active documents. Please activate at least two documents using the <a href="./manage">Manage</a> tool or <a href="./upload">upload</a> a new document.'
```

Comment:
use `url_for` instead of hard code url

Reply:
Using url_for links to a 404 page when clicking the upload or manage links.

Reply:
`<a href="{{ url_for('manage.manage') }}">Manage</a>` gives you are 404? Are you sure?

Reply:
Yes.

Reply:
What about other places where we use `<a href="{{ url_for('manage.manage') }}">Manage</a>`

Reply:
Can you check the generated html (the html that is displayed in the browser) to see if the generated html has the correct url?

Reply:
Weiqi, please check if you can reproduce this error.

Reply:
@danny98m I think you got this working while refactoring cutter? Could you please come back and take a look at this file again?

Reply:
It still does not work in the refactoring of cutter branch using url_for (still links to 404). The only reason the error message works in this here branch is because I made onclick functions.

Reply:
I think we were having trouble using Jinja template in the JS file. And in general we agree that using template in JS is not recommended. To avoid this usage, one possible solution is to save the URL in a ```<meta>``` tag. Refer to 58adef6 for details.

Reply:
Ah this makes a lot of sense.